vaccine-standby/functions/setup.js

```diff
@@ -0,0 +1,507 @@
+          {
+            event: 'deliveryFailure',
+          },
+        ],
+        properties: {
+          offset: {
+            x: 300,
+            y: 2950,
+          },
+          service: '{{trigger.message.InstanceSid}}',
+          channel: '{{trigger.message.ChannelSid}}',
+          from: '{{flow.channel.address}}',
+          body: 'What is your email address?',
+          timeout: '3600',
+        },
+      },
+      {
+        name: 'Goodbye',
+        type: 'send-message',
+        transitions: [
+          {
+            event: 'sent',
+          },
+          {
+            event: 'failed',
+          },
+        ],
+        properties: {
+          offset: {
+            x: 60,
+            y: 3460,
+          },
+          service: '{{trigger.message.InstanceSid}}',
+          channel: '{{trigger.message.ChannelSid}}',
+          from: '{{flow.channel.address}}',
+          to: '{{contact.channel.address}}',
+          body: "Thank you, you are now now on the vaccine standby list.\n\nWe will notify you when you're eligible to receive a vaccine.",
+        },
+      },
+      {
+        name: 'LanguagePreference',
+        type: 'send-and-wait-for-reply',
+        transitions: [
+          {
+            next: 'Goodbye',
+            event: 'incomingMessage',
+          },
+          {
+            event: 'timeout',
+          },
+          {
+            event: 'deliveryFailure',
+          },
+        ],
+        properties: {
+          offset: {
+            x: 60,
+            y: 3210,
+          },
+          service: '{{trigger.message.InstanceSid}}',
+          channel: '{{trigger.message.ChannelSid}}',
+          from: '{{flow.channel.address}}',
+          body: 'What is your preferred language?\n1. English\n2. Español\n3. 繁體中文\n\nReply"1", "2", or "3"',
+          timeout: '3600',
+        },
+      },
+      {
+        name: 'ZipCode',
+        type: 'send-and-wait-for-reply',
+        transitions: [
+          {
+            next: 'EssentialWorker',
+            event: 'incomingMessage',
+          },
+          {
+            event: 'timeout',
+          },
+          {
+            event: 'deliveryFailure',
+          },
+        ],
+        properties: {
+          offset: {
+            x: 360,
+            y: 1120,
+          },
+          service: '{{trigger.message.InstanceSid}}',
+          channel: '{{trigger.message.ChannelSid}}',
+          from: '{{flow.channel.address}}',
+          body: 'What is your 5-digit zip code?',
+          timeout: '3600',
+        },
+      },
+    ],
+    initial_state: 'Trigger',
+    flags: {
+      allow_concurrent_calls: true,
+    },
+  };
+
+  const client = context.getTwilioClient();
+
+  // Deploy Twilio Studio Flow
+  function deployStudio(formattedFlow) {
+    return new Promise((resolve, reject) => {
+      client.studio.flows
+        .create({
+          commitMessage: 'Code Exchange automatic deploy',
+          friendlyName: 'Vaccine Standby Intake',
```

Comment:
as I mentioned above this is friendlyName is going to be duplicated if someone re-deploys the app

Reply:
This is now resolved as I added a new `.env` variable called `FLOW_SID` that gets used to verify the existence of the studio flow instead of the friendly name. Thanks for pushing us to improve the architecture!